modules/db_instance/variables.tf

```diff
@@ -11,6 +11,16 @@ variable "storage_type" {
   default     = "gp2"
 }
 
+variable "storage_encrypted" {
+  description = "Specifies whether the DB instance is encrypted"
+  default     = false
+}
+
+variable "kms_key_id" {
+  description = "The ARN for the KMS encryption key. If creating an encrypted replica, set this to the destination KMS ARN."
```

Comment:
Add a note as you discovered like: `If storage_encrypted is set to true and kms_key_id is not specified the default KMS key created in your account will be used`.